README.md

```diff
@@ -46,34 +69,61 @@ You will need to have a native client application registered with Microsoft usin
+This project has adopted the 
+[Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). 
+For more information see 
+the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact 
+[opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
 
 ## Security Library
 
-This library controls how users sign-in and access services. We recommend you always take the latest version of our library in your app when possible. We use [semantic versioning](http://semver.org) so you can control the risk associated with updating your app. As an example, always downloading the latest minor version number (e.g. x.*y*.x) ensures you get the latest security and feature enhanements but our API surface remains the same. You can always see the latest version and release notes under the Releases tab of GitHub.
+This library controls how users sign-in and access services. We recommend you always take the 
+latest version of our library in your app when possible. We 
+use [semantic versioning](http://semver.org) so you can control the risk associated with updating 
+your app. As an example, always downloading the latest minor version number (e.g. x.*y*.x) ensures 
+you get the latest security and feature enhanements but our API surface remains the same. You 
+can always see the latest version and release notes under the Releases tab of GitHub.
 
 ## Security Reporting
 
-If you find a security issue with our libraries or services please report it to [secure@microsoft.com](mailto:secure@microsoft.com) with as much detail as possible. Your submission may be eligible for a bounty through the [Microsoft Bounty](http://aka.ms/bugbounty) program. Please do not post security issues to GitHub Issues or any other public site. We will contact you shortly upon receiving the information. We encourage you to get notifications of when security incidents occur by visiting [this page](https://technet.microsoft.com/en-us/security/dd252948) and subscribing to Security Advisory Alerts.
+If you find a security issue with our libraries or services please report it 
+to [secure@microsoft.com](mailto:secure@microsoft.com) with as much detail as possible. Your 
+submission may be eligible for a bounty through the [Microsoft Bounty](http://aka.ms/bugbounty) 
+program. Please do not post security issues to GitHub Issues or any other public site. We will 
+contact you shortly upon receiving the information. We encourage you to get notifications of when 
+security incidents occur by 
+visiting [this page](https://technet.microsoft.com/en-us/security/dd252948) and subscribing 
+to Security Advisory Alerts.
 
 
```

Comment:
What about having a ## More Information section with links to the ADAL Android conceptual documentation?